docs/topics/data_calculation.md

```diff
@@ -116,4 +116,17 @@ For those cases when the data values that want to be calculated are statistical
 the [Short Term Historic](../sth.md) can be used. This component stores historical information of Context Entities and can be used
 to retrieve statistical calculations over the stored data.
 
-// @GERMAN: SOME INFORMATION ABOUT HOW TO GET STATISTICS CALCULATED BY THE STH WOULD BE NEEDED
\ No newline at end of file
+As thoroughly detailed in the [STH documentation at ReadTheDocs](http://fiware-sth-comet.readthedocs.io/en/latest/index.html), the STH is able to calculate statistics about the evolution in time of certain entity attributes. To do it, there are 2 ways to notify this evolution in time to the component:
+
+1. Via the [Cygnus](https://github.com/telefonicaid/fiware-cygnus) component and subscribing it to the Context Broker instance.
+2. Via the [STH](https://github.com/telefonicaid/fiware-sth-comet) component itself and directly subscribing it to the Context Broker instance.
+
+Once the STH receives the notifications of entity attribute value changes, it is able to calculate and provide information about the mean, standard deviation, variance, maximum and minimum values (for numerical values) as well as the number of ocurrences (for textual values) for distinct resolutions or time frames (such as months, days, hours, minutes and seconds).
```

Comment:
This paragraph may be easier to read listing the options with bullets. Take into account that people usually make diagonal reads on this topics, so we better highlight the available options.